actioncable/CHANGELOG.md

```diff
@@ -1,2 +1,17 @@
+*   Add ability to detect a half-open connection
+
+    ActionCable doesn't have a mechanism to detect if it's connection is
```

Comment:
Typo: its.

Reply:
Fixed in the CHANGELOG file and the commit message. Thank you!